builds/win32/make_all.bat

```diff
@@ -62,8 +62,7 @@ for %%v in (gpre_boot build_msg codes) do (
 
 :: Firebird.conf, etc
 @copy %FB_GEN_DIR%\firebird.msg %FB_OUTPUT_DIR% > nul
-:: The line @UDF_COMMENT@ should be deleted from the target file.
-findstr /V "@UDF_COMMENT@" %FB_ROOT_PATH%\builds\install\misc\firebird.conf.in > %FB_OUTPUT_DIR%\firebird.conf
+@copy %FB_ROOT_PATH%\builds\install\misc\firebird.conf.in %FB_OUTPUT_DIR%\firebird.conf
 @copy %FB_ROOT_PATH%\builds\install\misc\databases.conf.in %FB_OUTPUT_DIR%\databases.conf >nul
```

Comment:
I'm just wondering, if these two files are not changed during the packaging, why they carry the .in suffix? Shouldn't they be renamed to plain .conf?

Reply:
Yes, should be renamed - but I suggest to do it after accepting this PR.
In order not to overload it.